.github/workflows/auto-release.yml

```diff
@@ -13,4 +13,38 @@ jobs:
     name: Automated release
     runs-on: ubuntu-latest
     steps:
-      - run: echo "Triggered successfully"
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Ensure we are checked out on the develop branch
+          ref: develop
+          # Ensure custom credentials are used when pushing
+          persist-credentials: false
+          # Fetch everything so we can checkout master
+          fetch-depth: 0
+      - name: Update simple-icons and package version
+        id: update
+        run: node scripts/version.js
```

Comment:
Not sure what we would gain by doing that 🤔 The latter _should_ never fail (assuming it worked in the past), so if the job fails its very likely something went wrong with this script.

Reply:
If a week there is no updates in `simple-icons`, the script would raise "Version already exists" error. In that case, the workflow will be marked as failed, but shouldn't be marked the commit part as skipped, or I'm wrong and will not be marked as failed?

Reply:
After a step in a job has failed, all subsequent steps are always skipped, see e.g. https://github.com/simple-icons/simple-icons-font/runs/1583076240?check_suite_focus=true

Reply:
Yes, but it's marked as failed not? Maybe we should difference between this expected failure (release not triggerring) and a unexpected failed (another error)?

Reply:
That seems like overkill to me, we also don't do `npm ci` in a separate job because it may fail (e.g. connection failure or invalid `package.json`) 🤔

Reply:
Okay, it's an edge case so no worries :+1: